.github/workflows/shared_meterpreter_acceptance.yml

```
@@ -185,9 +185,9 @@ jobs:
       - name: Checkout metasploit-framework commit
         uses: actions/checkout@v4
         with:
-          repository: rapid7/metasploit-framework
+          repository: zeroSteiner/metasploit-framework
```

I believe this should point at your forked repo and not Spencers:
```suggestion
          repository: dledda-r7/metasploit-framework
```